Sources/Plasma/Apps/plClient/Mac-Cocoa/NSString+StringTheory.mm

```diff
@@ -47,7 +47,7 @@ @implementation NSString (StringTheory)
 
 + (id)stringWithSTString:(const ST::string&)string
 {
-    return NSStringCreateWithSTString(string);
+    return hsAutorelease(NSStringCreateWithSTString(string));
```

Comment:
I'm leaning towards this file just turning off ARC instead of adding hsAutorelease.